### ISSUE TITLE
Failed to build iOS app

### ISSUE THREAD

**Issue by UGreek** [juliuscanute/qr_code_scanner#35]
Launching lib/main.dart on iPhone Andrey in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: WRYNMB2729
Running pod install...
Running Xcode build...
Xcode build done.                                            4,7s
Failed to build iOS app
Error output from Xcode build:
↳
    2019-08-18 10:17:42.711 xcodebuild[36516:1089909]  DTDeviceKit: deviceType from 1b083cf7ccc9ab170d1f15ea75cdc91d161ebc5c was NULL
    2019-08-18 10:17:42.765 xcodebuild[36516:1089913]  DTDeviceKit: deviceType from 1b083cf7ccc9ab170d1f15ea75cdc91d161ebc5c was NULL
    ** BUILD FAILED **

Xcode's output:
↳
    warning: The iOS deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.4.99. (in target 'Flutter')
    /Users/andrey/.pub-cache/hosted/pub.dartlang.org/qr_code_scanner-0.0.12/ios/Classes/FlutterQrPlugin.m:2:9: fatal error: 'qr_code_scanner/qr_code_scanner-Swift.h' file not found
    #import "qr_code_scanner/qr_code_scanner-Swift.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    Runner-hkmsyoepzonlwvejomucogedgxxm
    note: Using new build systemnote: Planning buildnote: Constructing build description

Could not build the precompiled application for the device.

It appears that your application still contains the default signing identifier.
Try replacing 'com.example' with your signing id in Xcode:
  open ios/Runner.xcworkspace

Error launching application on iPhone Andrey.

**Comment by singerbj** [comment]
Also having this error! Plz help!

**Comment by bissenj22** [comment]
This took care of it for me:
https://github.com/flutter/flutter/issues/22123

**Comment by juliuscanute** [comment]
Thanks to @bissenj22 👍

**Comment by qimingweng** [comment]
I'm still getting the issue, I'm not sure it's the XCode legacy build system issue.

I'm looking for `qr_code_scanner-Swift.h` in the project files and don't see any in the git repo. Is it possible this is pointing to something that has been renamed?

**Comment by qimingweng** [comment]
Update, OK, dug around a bit.

TLDR, I needed to make sure, `use_frameworks!` was not commented out in `Podfile`

```ruby
# ...

target 'Runner' do

  # ...

  use_frameworks!
  use_modular_headers!

# ...
```

I guess this makes sense because this package builds to a framework, and `FlutterQrPlugin.m` is just trying to import from that built framework.
